tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Looks good, TS docs are pretty great, but I've also found [this video](https://www.youtube.com/watch?v=eJXVEju3XLM&t=1s) helping

Reply:
Amazing, very good resource, thanks!